ts/__e2e__/app.e2e.ts

```diff
@@ -109,5 +117,53 @@ describe("e2e app", () => {
         .toExist()
         .withTimeout(WAIT_TIMEOUT_MS);
     });
+
+    it("should let the user pay a notice", async () => {
+      await element(by.text(I18n.t("global.navigator.wallet"))).tap();
+      await element(by.text(I18n.t("wallet.payNotice"))).tap();
+
+      await element(by.text(I18n.t("wallet.QRtoPay.setManually"))).tap();
+
+      const matchNoticeCodeInput = by.id("NoticeCodeInput");
+      await waitFor(element(matchNoticeCodeInput))
+        .toExist()
+        .withTimeout(WAIT_TIMEOUT_MS);
+
+      await element(matchNoticeCodeInput).replaceText("123123123123123123");
+      await element(by.id("EntityCodeInput")).replaceText("12345678901");
+
+      await element(by.text(I18n.t("global.buttons.continue"))).tap();
+      // await element(by.label("Continua")).tap();
+
+      await waitFor(element(by.label(I18n.t("wallet.continue"))))
+        .toExist()
+        .withTimeout(WAIT_TIMEOUT_MS);
+      await element(by.label(I18n.t("wallet.continue"))).tap();
+
+      const matchConfirmPayment = by.text(
+        `${I18n.t("wallet.ConfirmPayment.goToPay")} ${formatNumberCentsToAmount(
+          2322,
+          true
+        )}`
+      );
```

Comment:
My bad, I should have mentioned it: you need to use the `scripts/api-config.json` file for the dev-server to keep amounts consistent across the test.
I normally favour `text` and `label` over `testID` because it mimicks better how users identify the buttons on the UI.

Reply:
I already used the `api-config.json`, but this pr doesn't include any new changes to that file or I missing something? Maybe should include some new configuration?
Btw I agree with the preference of using `text` over `testID` when possible :D

Reply:
![facepalm](https://media.giphy.com/media/yvBAuESRTsETqNFlEl/giphy.gif)

Forgot to include it 😓  here we are e19aea1

Reply:
In order to have the test run I've also added `"allowRandomValues" : false`  in the `global` section of the config. I think we should add this also in the `scripts/api-config.json`, right?

Reply:
It should make any difference since it is not used [in my PR](https://github.com/pagopa/io-dev-api-server/pull/90/files) for supporting those new params and is apparently [yet to be supported](https://github.com/pagopa/io-dev-api-server/blob/master/src/config.ts#L79).

Reply:
uhm without using `"allowRandomValues" : false` the test keep failing because the amount changes every time, this is an example:

https://user-images.githubusercontent.com/26501317/141160557-0ceab01f-d9ee-456e-b92b-28af20adf565.mov

Using `"allowRandomValues" : false` I always obtain the fixed amount.  